scripts/bundleSize.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Size limit for all bundles used by each service (K)
+# Keep these +/- 5K and update frequently!
+min=684
+max=702
+
+services=( "news" "persian" "igbo" "yoruba" "pidgin" )
+failure=false
+
+for service in "${services[@]}"
```

Comment:
```suggestion
for service in ${services[@]}
```

The variable doesn't need string notation